Streams/MemoryStreamReader.h

```diff
@@ -18,7 +19,7 @@ class MemoryStreamReader : public SeekableStreamReader {
 	void ReadImplementation(void* buffer, std::size_t size) override;
 
 private:
-	char* streamBuffer;
-	size_t streamSize;
-	size_t position;
+	const char* streamBuffer;
```

Comment:
If you want to be really pedantic here, you can use:
```
const char* const streamBuffer;
```
That makes the pointer itself `const`, not just the data being pointed to.